src/styles/main.scss

```diff
@@ -1 +1,79 @@
-// put code here
+body {
+  margin: 0;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 10px;
+}
+
+
+.watch {
+  box-sizing: border-box;
+  width: 80vmin;
+  height: 80vmin;
+  border-radius: 50%;
+  border: 3px dashed black;
+  position: relative;
+
+  &__center {
```

Comment:
add bigger z-index to a center than hands have
<img width="114" alt="Screenshot 2024-01-08 at 15 38 05" src="https://github.com/mate-academy/layout_stop-watch/assets/117587463/d5acde90-2590-45a2-9ca9-b2bda60d0596">

Reply:
Adding Z-index to the center doesn't change it. I have tried putting z-index: 5 to the center and smaller indexes to hands, but it doesn't affect. Thus, I would be greatful for other suggestions.